textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/join/Join.java

```diff
@@ -70,9 +70,12 @@ public Join(IOperator outerOperator, IOperator innerOperator, JoinPredicate join
 
 	@Override
 	public void open() throws Exception, DataFlowException {
-		if (!(joinPredicate.getjoinAttribute().getFieldType().equals(FieldType.STRING)
-				|| joinPredicate.getjoinAttribute().getFieldType().equals(FieldType.TEXT))) {
-			throw new Exception("Fields other than \"STRING\" and \"TEXT\" are not supported by Join yet.");
+		if (!(joinPredicate.getjoinAttribute().getFieldType()
```

Comment:
Just curious, why does github show this difference?  If it's just formatting changes, let's try to avoid such changes.